src/formElementPlugins/search/src/ApplicationView.tsx

```diff
@@ -243,6 +272,7 @@ export const DisplaySelection: React.FC<DisplayProps> = ({
             {subtitle && (
               <Card.Meta>
                 <Markdown text={substituteValues(subtitle, item)} semanticComponent="noParagraph" />
+                SUBTITLE
```

Comment:
Hmm do we already have a localisation file for this plugin?

Reply:
This is a mistake -- sorry, I'll put it back to how it was.

Reply:
But yes, there is a localisation file for this one. 

Removed string: a4449590